Sources/Commands/SwiftTool.swift

```diff
@@ -366,6 +357,19 @@ public class SwiftTool<Options: ToolOptions> {
         }
     }
 
+    func buildPlan(graph: PackageGraph, config: Build.Configuration) throws -> BuildPlan {
```

Comment:
We don't need to pass config, it should be already available to this class via `options.config`